### ISSUE TITLE
production only? 

### ISSUE THREAD

**Issue by chrishough** [gshutler/browser_details#1]
Have you considered add a mode for this gem? something where we could only include it in production, but not development?

**Comment by gshutler** [comment]
What's the harm in including it in development logs?

**Comment by chrishough** [comment]
if you primarily work in a single browser, i.e. chrome you would be just cluttering up your logs.  after noticing the clutter in mine, i figured it would not be a bad idea for future versions.  thoughts?

**Comment by gshutler** [comment]
I'll take a look, I may not be able to get to it until the weekend though. I assume you're using Rails as nothing else configures it automatically?

In the meantime you could [delete it from the middleware within your development configuration the long winded way](http://guides.rubyonrails.org/rails_on_rack.html#middleware-stack-is-an-enumerable).

**Comment by chrishough** [comment]
@gshutler yes, I am using rails 4.  As of right now, I just pulled out the gem.  I was going to fork it but I was not sure of your roadmap. No rush, it was just an enhancement request b.c I wanted to keep using it.
